### PR TITLE
Update password-policy-recommendations.md

### PR DESCRIPTION
**Pull request by msbemba**
Added detail stating that password policy for Cloud only accounts cannot be changed and The only item you can change is how many days until a password expires and whether or not passwords expire at all.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7727

File: microsoft-365/admin/misc/password-policy-recommendations.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ description: "Make your organization more secure against password attacks, and b
 # Password policy recommendations
 
 As the admin of an organization, you're responsible for setting the password policy for users in your organization. Setting the password policy can be complicated and confusing, and this article provides recommendations to make your organization more secure against password attacks.
+
+For Cloud Only Accounts Microsoft has a pre-defined password policy that can't be changed. The only item you can change is how many days until a password expires and whether or not passwords expire at all.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/password-policy-recommendations.md]:
```suggestion
Microsoft cloud-only accounts have a pre-defined password policy that cannot be changed. The only items you can change are the number of days until a password expires and whether or not passwords expire at all. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder.

**Pull request merged**